docs/user/dql/functions.rst

```diff
@@ -2557,6 +2557,30 @@ Example::
     +------------------------------------+---------------------------------------+
 
 
+WEEKOFYEAR
+----------
+
+Description
+>>>>>>>>>>>
+
+The weekofyear function is a synonym for the `week`_ function.
```

Comment:
Is the underscore in `'week'_` a typo?

Reply:
No. It's a way for the documentation to create a link. When viewing the documentation it automatically creates a link to the appropriate function like it does [here](https://github.com/opensearch-project/sql/blob/main/docs/user/dql/functions.rst#week_of_year)